pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/CoalesceTransformFunction.java

```diff
@@ -31,21 +31,20 @@
 import org.roaringbitmap.RoaringBitmap;
 
 
-/**
- * The <code>CoalesceTransformFunction</code> implements the Coalesce operator.
- *
- * The result is first non-null value in the argument list.
- * If all arguments are null, return null.
- *
- * Note: arguments have to be compatible type.
- * Number of arguments has to be greater than 0.
- *
- * Expected result:
- * Coalesce(nullColumn, columnA): columnA
- * Coalesce(columnA, nullColumn): columnA
- * Coalesce(nullColumnA, nullColumnB): null
- *
- */
+///
+/// <code>CoalesceTransformFunction</code> implements the SQL COALESCE operator.
```

Comment:
Same for other places
```suggestion
/// [CoalesceTransformFunction] implements the SQL COALESCE operator.
```